back-end/api/quickstart/tests/endpoints/test_public_post_list.py

```diff
@@ -25,15 +25,21 @@ def setUp(self):
     self.post4 = Post.objects.create(
       **get_test_post_fields(4, visibility="Private"), author=self.author2
     )
```

Comment:
nit; we probably should have changed this before but maybe set unlisted=False on all these previous posts to help make it more explicit which post should be returned in the test.

As it is right now, we ensure the response has 3 posts returned but it's not exactly clear which ones are gonna be returned esp. cause these posts don't have unlisted explicitly set

Reply:
Thanks, done. It showed that the new post I added was the same as the first one, which was redundant so I removed it.